src/features/add-lesson/add-lesson.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import classNames from 'classnames';
+import faker from 'faker';
+import styles from './add-lesson.scss';
+
+export const AddLesson = () => {
+  const names = [
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+    `${faker.name.firstName()} ${faker.name.lastName()}`,
+  ];
```

Comment:
Please, could you do "git checkout dev" "git pull" and merge dev into your current branch?